libs/SalesforceSDK/res/xml/legacy_authenticator.xml

```diff
@@ -0,0 +1,3 @@
+<account-authenticator xmlns:android="http://schemas.android.com/apk/res/android"
+    android:accountType="com.salesforce.androidsdk" android:icon="@drawable/sf__icon"
```

Comment:
Hard coded to our default account type so that we can still retrieve the accounts to migrate when the app provides a new one.